src/main/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtil.java

```diff
@@ -83,15 +83,15 @@ private RefDataUtil() {
 
     private static final String DEFAULT_ORG_PROFILE_ID = "SOLICITOR_PROFILE";
     private static final Map<String, List<String>> ORG_TYPE_TO_ORG_PROFILE_IDS = Map.ofEntries(
-            new SimpleEntry<String, List<String>>("SOLICITOR_ORG", List.of("SOLICITOR_PROFILE")),
-            new SimpleEntry<String, List<String>>("LOCAL_AUTHORITY_ORG", List.of("SOLICITOR_PROFILE")),
-            new SimpleEntry<String, List<String>>("OTHER_ORG", List.of("SOLICITOR_PROFILE")),
-            new SimpleEntry<String, List<String>>("OGD_DWP_ORG", List.of("OGD_DWP_PROFILE")),
-            new SimpleEntry<String, List<String>>("OGD_HO_ORG", List.of("OGD_HO_PROFILE")),
-            new SimpleEntry<String, List<String>>("OGD_OTHER_ORG", List.of("SOLICITOR_PROFILE"))
+            new SimpleEntry<String, List<String>>("SOLICITOR-ORG", List.of("SOLICITOR_PROFILE")),

```

Comment:
Ive not confirmed the hyphen is allowed yet - be aware this setup may change

Reply:
If I revert back to underscores, the integration test fails because they will fail validation. Do we merge in as is and visit back at a later date?